### ISSUE TITLE
Hi! I made voronoi diagram version of cellauto :)

### ISSUE THREAD

**Issue by luckylooke** [sanojian/cellauto#6]
Hello, I haven't found another way how to contact you, github should implement at least personal messages to users :wink: 

I have rewrite your cellauto.js to cellautoVoronoi.js for special use case where Voronoi diagram is used instead of grid.

You can see live [DEMO](http://luckylooke.github.io/phaser-islandjs-plugin/), just click on cave icon in first generated island, every click then is one step. Codes are at the bottom of page.

Let me know what do you think. Thanks! :smile:

**Comment by sanojian** [comment]
Hey that is cool.  I am having a hard time coming up with a practical use for it though.  Can you think of any?

**Comment by luckylooke** [comment]
Yeah I will come with game using it but it will take some time ;) I will let you know then, for sure :+1:
